annotation-processor/src/main/java/com/linecorp/armeria/server/annotation/processor/DocumentationProcessor.java

```diff
@@ -73,7 +77,12 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
             try {
                 writeProperties(className, properties);
             } catch (IOException e) {
-                e.printStackTrace();
+                final StringWriter writer = new StringWriter();
+                e.printStackTrace(new PrintWriter(writer));
+                processingEnv.getMessager().printMessage(
+                        Kind.WARNING,
+                        String.format("Could not write properties for: %s\n%s",
```

Comment:
Let's rephrase a little bit for consistency:
`Failed to write properties for: ...`
Also, how about using `System.lineSeparator()`?

Reply:
Java handles platform specific things somewhat inconsistent... for the old io api it didn't matter if you used `/` or `\` on windows but here it does...

Reply:
IIRC you can just use `%n`.

Reply:
Didn't know about it. 😄